### PR TITLE
Make fungal infections less obvious to player

### PR DESCRIPTION
**Pull request by connorpmartin**
#### Summary
None

#### Purpose of change

Fungal infections are immediately visible to the player, since the status is displayed on the speed modification bar.
This isn't ideal: the presence of an infection should be obfuscated until the player starts coughing up mushrooms.

#### Describe the solution

Change the debuff name to "tingling sensation," to match the fungus removal message.

#### Describe alternatives you've considered

A different name for the debuff which is still thematically appropriate.

#### Testing

Before:
image
After:
image

### CODE REVIEW

**Comment by actual-nh** [comment]
Ping: @Venera3?

**Comment by Maleclypse** [comment]
I would review the planned overhaul of fungus as discussed https://www.reddit.com/r/cataclysmdda/comments/ni8h2b/fungaloid_analysis_and_possible_solutions_to_long/ and probably some on dev discord as well.   I'm not following it closely so I don't know for sure if your change interacts with the plans going forward.

**Comment by connorpmartin** [comment]
I'm assuming Venera3 is working on a fungal rebalance, so if they have some insight on whether they intend the disease to be visible or not that'd be great.
Right now, the disease is half-visible: it appears on the speed modification but not the status bar. It's probably best to either make fungal infections fully visible or obfuscate the speed modifier, and that's up to whoever's reworking fungus.

**Comment by Venera3** [comment]
I don't have any concrete plans for the Mycus apart from some vague ideas making fungicide less of a one-shot thing and turning it uncraftable (bleach plus bleach make no caspofungin). There's a discussion to be had what's the rationale behind terrestrial fungicides hurting the insidious transdimensional eldritch god at all, but I digress.

Regarding the visibility, there sadly isn't really a clear line of what information the player should or shouldn't have access to - the speed readout is notoriously open, with iirc no option to hide maluses at all. I'm personally all for obfuscating health information, if for no other reason to give some semblance of a reason to have the blood analysis cbm installed.

File: data/json/effects.json [hunk 1]
```diff
@@ -1041,7 +1041,7 @@
     "apply_memorial_log": "Contracted a fungal infection.",
     "remove_memorial_log": "Cured the fungal infection.",
     "miss_messages": [ [ "You feel sick inside.", 1 ] ],
-    "speed_name": "Fungal infection",
+    "speed_name": "Tingling Sensation",
```
Comment (Night-Pryanik) [inline data/json/effects.json:1044]:
```suggestion
    "speed_name": "Tingling sensation",
```

**Comment by Kelenius** [comment]
So this doesn't actually hide anything, just changes the name to something less clear. This will only serve to make the game even less newbie-friendly.

**Comment by SariusSkelrets** [comment]
It will need other causes of tingling sensation for the change to be meaningful 

If there’s only one thing that gives it, the cause is obvious 

I think that a itch might work better: it’s also used by dermatiks and kaluptic psychosis so the cause is uncertain

**Comment by actual-nh** [comment]
What is the reasoning behind the effects? Is this infection happening in the skin (as it appears in the above and from the spore effect description), the lungs ("coughing up mushrooms"), both, or what?

**Comment by SariusSkelrets** [comment]
> What is the reasoning behind the effects? Is this infection happening in the skin (as it appears in the above and from the spore effect description), the lungs ("coughing up mushrooms"), both, or what ("you feel sick inside" as a miss message indicates that some internal sensations are off)?

Since the infection starts by the skin, I thought that the early phases could cause an itch

The fungal infection currently affects various and unrelated areas of the body during the spread: it starts by the skin, visits the mouth (mushroom taste), then the digestive system (vomit grey goo) before focusing on the arms and lungs (broken arm and coughing spores)

The symptoms might need to be revisited: any veteran player not avoiding the fungus will start looking for antifungals at the instant a shroomy taste is mentioned. A ‘’weird taste’’ is less specific and can be used for other effects

**Comment by actual-nh** [comment]
> > What is the reasoning behind the effects? Is this infection happening in the skin (as it appears in the above and from the spore effect description), the lungs ("coughing up mushrooms"), both, or what ("you feel sick inside" as a miss message indicates that some internal sensations are off)?
> 
> Since the infection starts by the skin, I thought that the early phases could cause an itch

OK.

> The fungal infection currently affects various and unrelated areas of the body during the spread: it starts by the skin, visits the mouth (mushroom taste), then the digestive system (vomit grey goo) before focusing on the arms and lungs (broken arm and coughing spores)

Umm... broken arm?!?

> The symptoms might need to be revisited: any veteran player not avoiding the fungus will start looking for antifungals at the instant a shroomy taste is mentioned. A ‘’weird taste’’ is less specific and can be used for other effects

Anyone happen to know what having thrush (fungal infection of the mouth/throat, basically) tastes like?

**Comment by SariusSkelrets** [comment]
> Umm... broken arm?!?

It autocorrected. It’s supposed to be both arms that are broken by the fungus

> Anyone happen to know what having thrush (fungal infection of the mouth/throat, basically) tastes like?

I don’t know: I’m just telling the current effects. I’m for removing the obvious taste or changing it for something generic

**Comment by actual-nh** [comment]
> > Umm... broken arm?!?
> 
> It autocorrected. It’s supposed to be both arms that are broken by the fungus

That's not better.

> > Anyone happen to know what having thrush (fungal infection of the mouth/throat, basically) tastes like?
> 
> I don’t know: I’m just telling the current effects. I’m for removing the obvious taste or changing it for something generic

I understand; I was talking about IRL. @Venera3, @I-am-Erk?

**Comment by I-am-Erk** [comment]
For fungal infections alone this seems fine, in terms of larger fungal planning, orenaudeles and I had some good ideas that I could track down and make into a discussion thread. Basically fungal blooms and towers would be control centers, and the further away from them one got the slower the fungus would spread.

**Comment by connorpmartin** [comment]
> So this doesn't actually hide anything, just changes the name to something less clear. This will only serve to make the game even less newbie-friendly.

A good number of people play blind without looking anything up on a wiki.
Frankly, even if we hid the status altogether there are only so many hidden 10% speed statuses out there, so it would still be easy to find for a determined player. This would still work to preserve a factor of surprise / realism for players who don't want to use a wiki.

**Comment by SariusSkelrets** [comment]
Don’t forget to change the remove_message, it’s still about the tingling sensation

**Pull request merged**